src/themes/dojo/slidePane.m.css

```diff
@@ -1,34 +1,91 @@
 @import './variables.css';
 
+.root,
+.root * {
+	box-sizing: border-box;
+}
+
 .underlayVisible {
 	background: var(--underlay-bg);
 }
 
-.content {
-	font: var(--base-font);
+.pane {
+	background: var(--dojo-white);
+	border: var(--border-thin) solid var(--border-color);
+	box-shadow: var(--drop-shadow);
+	box-sizing: border-box;
+	font-size: var(--base-font-size);
 	height: 100%;
+	line-height: var(--base-line-height);
 	position: fixed;
 	top: 0;
-	background-color: var(--title-bar-bg);
-	color: var(--title-bar-color);
-	padding: var(--container-padding);
-	box-sizing: border-box;
+}
+
+.content {
+	padding: calc(2 * var(--grid-size));
 }
 
 .left {
 	left: 0;
-	transform: translateX(-100%)
+	transform: translateX(calc(-100% - 10px))
 }
 
 .right {
 	right: 0;
-	transform: translateX(100%);
+	transform: translateX(calc(100% + 10px));
 }
 
 .open {
 	transform: translateX(0%);
 }
 
+.close {
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%);
+	right: 35px;
+	cursor: pointer;
+	font-size: 0;
+	border: none;
+	background: none;
+}
+
+.close:after {
+	content: '✕';
+	display: block;
+	font-size: 14px;
+}
+
+.title {
+	border-bottom: var(--border-thin) solid var(--border-color);
+	box-sizing: border-box;
+	flex: 0 0 auto;
+	font-size: var(--header-font-size);
+	height: calc(7 * var(--grid-size));
+	line-height: var(--header-line-height);
+	padding: calc(2 * var(--grid-size));
+	position: relative;
+	text-align: left;
+}
+
+.close {
+	background: none;
+	border: none;
+	cursor: pointer;
+	font-size: 0;
+	padding: var(--grid-size);
+	position: absolute;
+	right: calc(2 * var(--grid-size));
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.close:after {
+	content: '✕';
```

Comment:
doesn't need to be now, but at some point this should probably be an icon (I think it is in the designs)